genprotocol/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "kbfs-protocol",
```

Comment:
Looks like this file is copy-pasted from `kbfs-server`.  Can you customize it for search please?